src/main/java/org/team1540/robot2024/subsystems/drive/Module.java

```diff
@@ -51,6 +57,14 @@ public Module(ModuleIO io, int index) {
 
         turnFeedback.enableContinuousInput(-Math.PI, Math.PI);
         setBrakeMode(true);
+
+        driveMotorDisconnected =
```

Comment:
What do we want to constitute an error vs a warning?

Reply:
The comments from the code that I was "inspired" by say that an error is 
>for problems which will seriously affect the robot's functionality and thus require immediate attention.
and a warning is 
>for problems which could affect the robot's functionality but do not necessarily require immediate attention.

From the way that I've seen it used in other implementations, a warning is something a single device being disconnected, while an error is something critical like an entire can bus dying.